app/helpers/data_getter.py

```diff
@@ -94,7 +94,7 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
+        return Event.query.order_by(desc(Event.id)).filter_by().all()
```

Comment:
@saptakS In the Super Admin section, `all_ticket_stats` variable is supposed to store the information about the tickets of all the events including the trashed ones. Since the get_all_events() function didn't returned the trashed events, the app crashed when an event was deleted in admin section. So I modified the definition of the function.

I looked at the code and there seems to be many places where this function is called. I will change it to its original definition and concat the `all_events` and `trash_events` when the need for all the events including the trashed events arises. Thank you so much for pointing out the mistake. :)